widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1189,6 +1189,11 @@ button#schedule-dialog-select-button {
   border-right: solid 1px transparent;
 }
 
+.pentaho-tabWidget:focus-visible {
+  outline: 3px solid #0075ff;
+  outline-offset: -2px;
+}
+
```

Comment:
This is the default focus ring style. Should not be needed. See https://github.com/hv-vamsi/pentaho-commons-gwt-modules/blob/a1828a129741e00f4f67a504d1076c04baadd315/widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css#L43.